.travis.yml

```diff
@@ -1,10 +1,5 @@
 language: erlang
-notifications:
-  email: knutin@gmail.com
 otp_release:
-   - 17.0
-   - R16B03-1
-   - R16B03
-   - R16B02
-   - R16B01
-   - R16B
+    - 21.0
```

Comment:
Elixir 1.7.3 (used in QT) is based on OTP 19, gacollect uses 21.1. Might be worth including these versions too.

Reply:
ok, I will add those